src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -85,15 +85,15 @@ export class CalciteRadioButton {
     this.input.required = required;
   }
 
-  /** The scale (size) of the radio button.  <code>scale</code> is passed as a property automatically from <code><calcite-radio-button-group></code>. */
+  /** The scale (size) of the radio button.  <code>scale</code> is passed as a property automatically from <code>calcite-radio-button-group</code>. */
```

Comment:
I believe these are interpreted as markdown, so you could theoretically use backticks instead of `code`?